web/core/components/analytics/custom-analytics/select/project.tsx

```diff
@@ -25,7 +25,7 @@ export const SelectProject: React.FC<Props> = observer((props) => {
       content: (
         <div className="flex items-center gap-2 truncate">
           <span className="text-[0.65rem] text-custom-text-200 flex-shrink-0">{projectDetails?.identifier}</span>
-          <span className="flex-grow truncate">{projectDetails?.name}</span>
+          <span className="flex-grow w-[192px] truncate">{projectDetails?.name}</span>
```

Comment:
Why is it 192px?

Reply:
Also, we are using truncate at 3 places. See if we need three, and which of them actually works.